src/promptflow-evals/promptflow/evals/evaluators/_chat/_chat.py

```diff
@@ -287,3 +235,75 @@ def _validate_conversation(self, conversation: List[Dict]):
         # Ensure the conversation ends with an assistant's turn
         if expected_role != "user":
             raise ValueError("The conversation must end with an assistant's turn.")
+
+
+class ChatEvaluator:
+    """
+    Initialize a chat evaluator configured for a specific Azure OpenAI model.
+
+    :param model_config: Configuration for the Azure OpenAI model.
+    :type model_config: AzureOpenAIModelConfiguration
```

Comment:
nit: for reference documentation, we want to use the full path for any internal objects.

i.e. `~promptflow.core.AzureOpenAIModelConfiguration`

Full documentation guide [here](https://github.com/microsoft/promptflow/blob/main/docs/dev/documentation_guidelines.md)

Reply:
Got it. Will address all those documentation comments in a follow up PR.